module-1/lab-functional-programming/your-code/Q2.ipynb

```diff
@@ -15,12 +15,22 @@
   },
```

Comment:
Let's review this afternoon because you have errors due to the paths of the files. 
The final print(bow) is not throwing a correct output.